include/audio/list.h

```diff
@@ -0,0 +1,43 @@
+#ifndef AUDIO_LIST_H
+#define AUDIO_LIST_H
+
+#include "PR/ultratypes.h"
+
+// A node in a circularly linked list. Each node is either a head or an item:
+// - Items can be either detached (prev = NULL), or attached to a list.
+//   'value' points to something of interest.
+// - List heads are always attached; if a list is empty, its head points
+//   to itself. 'count' contains the size of the list.
+// If the list holds notes, 'pool' points back to the pool where it lives.
+// Otherwise, that member is NULL.
+typedef struct AudioListItem {
+    /* 0x00 */ struct AudioListItem* prev;
+    /* 0x04 */ struct AudioListItem* next;
+    union {
+        /* 0x08 */ void* value; // either Note* or SequenceLayer*
+        /* 0x08 */ s32 count;
+               } u;
+    /* 0x0C */ struct NotePool* pool;
```

Comment:
The spacing here is weird.
What do you think about this instead?
```suggestion
    union {
        /* 0x08 */ void* value; // either Note* or SequenceLayer*
        /* 0x08 */ s32 count;
    } u;
    /* 0x0C */ struct NotePool* pool;
```